travvy/src/store/store.js

```diff
@@ -20,6 +22,10 @@ export default new Vuex.Store({
         },
         setSe(state,selectedAttraction){
             state.selectedAttraction = selectedAttraction
+        },
+        //when a user logs in, set the variable to their email
+        setUserEmail(state,userEmail){
+            state.userEmail = userEmail
```

Comment:
this is great, we can use this global variable when we are populating the rest of the user attributes on the Onboarding page, and in the future through the Profile page